Marek Piechocinski - Layout_1

marekpiechocinski opened the pull request.

marekpiechocinski commented:
First submission of Layout_1

lukasc-ubc commented:
Error: File 'simulations/EBeam_marekpiechocinski.gds' is not in the 'submissions' folder.

lukasc-ubc commented:
Hi Marek,

I deleted the file in the "simulations" folder.  I read through the error logs, and there a few things for you to fix:

1) there should only be ONE top cell per layout file.  Some have 2, some have 5.
2) you have many different files (9 in total), but they all seem similar, and incomplete. Please delete the ones that shouldn't be there, and only keep the file you want fabricated.
3) your layout has layer 69/0, which doesn't exist in our PDK
4) your layouts are missing the opt_in labels, required to test your design

marekpiechocinski commented:
Hi Lukas,

Thank you for checking my layouts. To resolve those errors I have produced
a fresh one which passed all the checks, and I removed all previous copies.

Also, I would like to thank your team for incorporating my late submission
to the main layout.

Kind regards,

Marek.



On Sat, 17 May 2025 at 06:10, Lukas Chrostowski ***@***.***>
wrote:

> *lukasc-ubc* left a comment (SiEPIC/openEBL-2025-05#173)
> <https://github.com/SiEPIC/openEBL-2025-05/pull/173#issuecomment-2888088861>
>
> Hi Marek,
>
> I deleted the file in the "simulations" folder. I read through the error
> logs, and there a few things for you to fix:
>
>    1. there should only be ONE top cell per layout file. Some have 2,
>    some have 5.
>    2. you have many different files (9 in total), but they all seem
>    similar, and incomplete. Please delete the ones that shouldn't be there,
>    and only keep the file you want fabricated.
>    3. your layout has layer 69/0, which doesn't exist in our PDK
>    4. your layouts are missing the opt_in labels, required to test your
>    design
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/SiEPIC/openEBL-2025-05/pull/173#issuecomment-2888088861>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BSQQD53XJN7AZ27DTZEOJ4T263AEZAVCNFSM6AAAAAB5JD4436VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDQOBYGA4DQOBWGE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
